src/SUMMARY.md

```diff
@@ -32,6 +32,7 @@
 - [Interrupts](./Interrupts.md)
   - [Interrupt Sources](./Interrupt_Sources.md)
 - [CGB Registers](./CGB_Registers.md)
+  - [CGB Infrared](./CGB_Infrared.md)
```

Comment:
This is a largely reasonable place, but what about grouping it with Serial instead?

Reply:
I think that'd only make sense if the CGB Registers document were to be targetted for splitting into respective sections in general (particularly HDMA and OPRI).

Reply:
Agreed on splitting HDMA, but OPRI, KEY1 (and KEY0 which was mysteriously omitted?) are definitely related to the CGB itself, imo, so keeping them grouped in this document makes sense.